### PR TITLE
Fixed single quotes of awk command

### LINKED ISSUE #15: Minor bug in stop-dashboard script
Hi SAPS developers,
I found that the script 'bin/stop-dashboard' is not properly working because of a minor typo in the awk command. You need to use this commas '' instead of ‘‘, so this error appears:

awk: cmd. line:1: ‘{print
awk: cmd. line:1: ^ invalid char '?' in expression

The right line is:
ps xau | grep app.js | awk '{print $2}' | xargs sudo kill

Regards,
Amanda.

### PR DESCRIPTION
**Pull request by wesleymonte**
Resolves #15

### CODE REVIEW

**Review by raonismaneoto — CHANGES_REQUESTED** [review]
can we trust in this script? I'm afraid that this grep command may cause some unexpected behavior, app.js is very common. You can search the PID by using `lsof -i :port | awk '{ print $2}'`, the port is the port where the application is running. If u have some time try it out and check if it works

File: bin/stop-dashboard [hunk 1]
```diff
@@ -1,3 +1,14 @@
 #!/bin/bash
 
-ps xau | grep app.js | awk '{print $2}' | xargs sudo kill
+readonly PORT=8081
+
+echo "Run as Sudo!"
```
Comment (raonismaneoto) [inline bin/stop-dashboard:5]: as root* or with sudo

**Review by raonismaneoto — CHANGES_REQUESTED** [review]

File: bin/stop-dashboard (hunk 1 above)
Comment (wesleymonte) [reply bin/stop-dashboard]: Done.

**Review by raonismaneoto — APPROVED** [review]

File: bin/stop-dashboard [hunk 2]
```diff
@@ -1,3 +1,14 @@
 #!/bin/bash
 
-ps xau | grep app.js | awk ‘{print $2}’ | xargs sudo kill
\ No newline at end of file
+readonly PORT=8081
+
+echo "Run as root!"
```
Comment (thiagomanel) [inline bin/stop-dashboard:5]: remove this echo. add as comment

File: bin/stop-dashboard [hunk 3]
```diff
@@ -1,3 +1,14 @@
...
+
+PID=$(lsof -ti:${PORT})
+
+if [ -z $PID ]; then
```
Comment (thiagomanel) [inline bin/stop-dashboard:9]: change line 10 to `echo "PID" ${PID} "not found"

**Review by thiagomanel — CHANGES_REQUESTED** [review]
one request to remove a line and another to improve echo

File: bin/stop-dashboard (hunk 2 above)
Comment (wesleymonte) [reply bin/stop-dashboard]: Done.

File: bin/stop-dashboard (hunk 3 above)
Comment (wesleymonte) [reply bin/stop-dashboard:9]: Done.

**Review by joivo — APPROVED** [review]

**Review by thiagomanel — APPROVED** [review]
lgtm

**Pull request merged**